### DISCUSSION TITLE
Why doesn't the repo have issues?

### DISCUSSION THREAD

**Discussion by huaracheguarache** [solokeys/solo2#124] [General]
Why does the repo not have issues in addition to discussions? I think it would be easier to keep track of bugs if they could be submitted as issues.

**Comment by huaracheguarache** [comment]
@conorpp Could you please consider enabling issues in the repo to keep bug reports a bit more organised?

**Reply by nickray** [reply]
This repository mostly assembles other libraries into a running firmware bundle.

- technical bug reports should go into issues of the corresponding library
- higher-level questions about using the device should go into a discussion in this repository
- customer service problems like "key seems broken" (or anything else if you feel you're not being heard on GitHub) should be directed to support@solokeys.com

We ask that users please check if related issues and discussions exist before filing new ones, to reduce discussion/issue fatigue. SoloKeys is still an unprofitable open source side business of its creators.

**Reply by KizzyCode** [reply]
Well, now we have the situation that the discussions are heavily used as issue replacement, which is just chaotic and confusing.

E.g. at least these discussions are completely valid issues and should have been treated as such:
 - https://github.com/solokeys/solo2/discussions/126
 - https://github.com/solokeys/solo2/discussions/108
 - https://github.com/solokeys/solo2/discussions/116
 - https://github.com/solokeys/solo2/discussions/113
 - https://github.com/solokeys/solo2/discussions/115
 - https://github.com/solokeys/solo2/discussions/90
 - https://github.com/solokeys/solo2/discussions/111
 - https://github.com/solokeys/solo2/discussions/106

I also want to state that it is of course completely fine to close/move issues that affect foreign repos or have a Q/A-character; however the current approach is highly uncommon for github and feels unsatisfying/opaque.

**Comment by needs-coffee** [comment]
There is a place for both issues and for discussions in a modern GitHub repo.
Discussions have really become prevalent as the issues were getting filled with non-issues/help/feature requests and obscuring actual issues but they are not a replacement.
The main problem with discussions is that they cant be 'closed'.
Ideally I think there should be issues open on this repo for issues global to the solo2, as some of the posts mentioned by @KizzyCode are. An example of a 'closed' discussion about this would be the first issue above (#126) about the release notes unclear - this has now been solved so should be marked as such, but cant be done so when it is in the form of a discussion.

An issue template could be used to direct users to the libraries where the issue should be filed if it pertains to one of the libraries not the key as a whole?
